src/libraries/System.ServiceProcess.ServiceController/src/System/ServiceProcess/ServiceBase.cs

```diff
@@ -473,8 +473,7 @@ private void DeferredPowerEvent(int eventType, IntPtr eventData)
             // already been freed.
             try
             {
-                PowerBroadcastStatus status = (PowerBroadcastStatus)eventType;
-                _ = OnPowerEvent(status);
+                _ = OnPowerEvent((PowerBroadcastStatus)eventType);
```

Comment:
```suggestion
                OnPowerEvent((PowerBroadcastStatus)eventType);
```

Reply:
When would you prefer discards?

Reply:
> When would you prefer discards?

For ignored return values, only if it's critical that someone reading the code understands data is being thrown away, or if the compiler complains.  The former might show up in one of our security libraries, for example.  The latter might show up with a method returning a Task in an async method and where we explicitly want to ignore it.